src/components/MessageChat.tsx

```diff
@@ -0,0 +1,104 @@
+import { IMessageType } from '../interface';
+import { useRecoilValue } from 'recoil';
+import { chatRoomState } from '../atom';
+import styled, { css } from 'styled-components';
+
+const MessageChat = ({ message }: { message: IMessageType }) => {
+  const { currentUser } = useRecoilValue(chatRoomState);
+  const isUser = message.user.id === currentUser.id;
+  console.log(isUser);
+  console.log(currentUser.id);
+  console.log(message.user.id);
+  return (
+    <MessageBox isUser={isUser}>
+      <section>
+        <MessageProfAll>
+          <MessageUser isUser={isUser}>{message.user.name}</MessageUser>
+          <Messageimg isUser={isUser} _id={message.user.id}></Messageimg>
```

Comment:
조금더 구체적인 네이밍이 좋을 것 같아요!
```suggestion
          <Messageimg isUser={isUser} messageId={message.user.id}></Messageimg>
```